Formula/assimp.rb

```diff
@@ -16,15 +16,27 @@ class Assimp < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "ninja" => :build
```

Comment:
Do we need to use Ninja?

Reply:
Can you elaborate a bit?

Reply:
Why not use `make` instead of `ninja`?

Reply:
let me give a try on that.

Reply:
actually did not make it work with `make`